sources/packages/web/src/components/layouts/aest/AESTHomeSideBar.vue

```diff
@@ -1,51 +1,68 @@
 <template>
-  <v-navigation-drawer app class="body-background" permanent>
-    <v-list density="compact" nav>
+  <v-navigation-drawer app v-model="drawer" color="background" permanent>
+    <v-list
+      active-class="active-sidebar-item"
+      density="compact"
+      bg-color="background"
+      active-color="primary"
+    >
       <v-list-item
         v-for="topItem in topItems"
         :key="topItem.label"
         :prepend-icon="topItem.icon"
         :title="topItem.label"
-        @click="topItem.command"
+        :to="topItem.command()"
       />
     </v-list>
-    <v-list density="compact" nav>
-      <v-list-subheader>Student requests</v-list-subheader>
+    <v-list
+      active-class="active-sidebar-item"
+      density="compact"
+      bg-color="background"
+      active-color="primary"
+    >
+      <v-list-subheader class="nav-subtitle">Student requests</v-list-subheader>
```

Comment:
The spacings in the left menu seem different from Figma, I am not sure if it is part of the PR or not, but can we have at least the proper case adjusted for the sub-headers, as shown below?

![image](https://user-images.githubusercontent.com/61259237/188204729-93623b1e-6167-4113-8cb5-ff57834823f2.png)

Reply:
Also the Search Student and Search Institution are singular in figma but plural here. 
![image](https://user-images.githubusercontent.com/54600590/188207644-95378999-89d7-4927-b3d2-f95fc7fc9055.png)

Reply:
Yes these changes has been verified by @hellolynn-tbtb and the figma was not current. @hellolynn-tbtb Please verify.

Reply:
institution sidebar subtitle font is different in Figma, maybe we can check with @hellolynn-tbtb 
![image](https://user-images.githubusercontent.com/77353155/188236429-15eee1cd-f525-498b-a655-2f4596022347.png)